app/js/components/common/header.js

```diff
@@ -0,0 +1,146 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+import React, {Component} from 'react';
+import {Link, IndexLink} from 'react-router';
+import {ApiHelper} from '../../helpers/apiHelper';
+
```

Comment:
Do you think we can easily create some sort of reusable component that even other andela apprentice groups can just reuse for this header?

Reply:
Yes that can help.